### PR TITLE
InstructorFeedbackEditPageUiTest failing on live server #5795

### LINKED ISSUE #5795: InstructorFeedbackEditPageUiTest failing on live server
I think a hard coded url is the culprit. V5.82
Failing at line `125         feedbackEditPage.verifyHtml("/instructorFeedbackEditEmpty.html");`

### PR DESCRIPTION
**Pull request by belyabl9**
Fixes #5795

### CODE REVIEW

File: src/test/java/teammates/test/driver/HtmlHelper.java [hunk 1]
```diff
@@ -413,6 +413,10 @@ private static String replaceUnpredictableValuesWithPlaceholders(String content)
                       // jQuery-ui js file
                       .replace(Const.SystemParams.getjQueryUiFilePath(TestProperties.isDevServer()),
                                "${lib.path}/jquery-ui.min.js")
+                      .replace(TestProperties.TEAMMATES_URL + "/js/lib/skins/lightgray/skin.min.css",
```
Comment (kanghj) [inline src/test/java/teammates/test/driver/HtmlHelper.java]: Let's keep the convention that the other replacements here have. Add a comment to describe what is getting replaced.

Comment (kanghj) [inline src/test/java/teammates/test/driver/HtmlHelper.java]: Also, let's test this in `GodModeTest`, an example for replacement and its expected replacement can be tested with the files `godmode.html`, `godmodeExpectedOutput.html`, and `godmodeExpectedPartOutput.html`

**Comment by kanghj** [comment]
Some comments added. I have verified that it works on a staging server

**Comment by belyabl9** [comment]
Ready to review

File: src/test/java/teammates/test/driver/HtmlHelper.java [hunk 2]
```diff
@@ -461,7 +467,11 @@ public static String injectContextDependentValuesForTest(String content) {
                       .replace("<!-- filepath.jquery -->",
                                Const.SystemParams.getjQueryFilePath(TestProperties.isDevServer()))
                       .replace("<!-- filepath.jquery-ui -->",
-                               Const.SystemParams.getjQueryUiFilePath(TestProperties.isDevServer()));
+                               Const.SystemParams.getjQueryUiFilePath(TestProperties.isDevServer()))
+                      .replace("<!-- tinymce.skin.min -->",
+                               "${lib.path}/skins/lightgray/skin.min.css")
```
Comment (kanghj) [inline src/test/java/teammates/test/driver/HtmlHelper.java]: Looking at the rest of the context dependent values, the value after replacement should be a value that actually occurs in actual usage. I don't think `${lib.path}` is substituted with the actual lib path value after this is executed in GodModeTest on `godmode.html`? This should be something like `TestProperties.TEAMMATES_URL  + "/js/lib/skins/lightgray/skin.min.css"`.

**Comment by belyabl9** [comment]
Ready to review

**Comment by kanghj** [comment]
@belyabl9, can you rebase or merge master to this branch and ping me?

**Comment by belyabl9** [comment]
@kanghj , rebased

**Comment by kanghj** [comment]
Thanks, @belyabl9

**Pull request merged**